satpy/modifiers/filters.py

```diff
@@ -0,0 +1,34 @@
+"""Tests for image filters."""
+import logging
+
+import xarray as xr
+
+from satpy.modifiers import ModifierBase
+
+logger = logging.getLogger(__name__)
+
+
+class Median(ModifierBase):
+    """Apply a median filter to the band."""
+
+    def __init__(self, median_filter_params, **kwargs):
+        """Create the instance.
+
+        Args:
+            median_filter_params: The arguments to pass to dask-image's median_filter function. For example, {size: 3}
+                                  makes give the median filter a kernel of size 3.
+
+        """
+        self.median_filter_params = median_filter_params
```

Comment:
Params versus kwargs? Just a though.

Reply:
I chose params... but if you want I can change

Reply:
Nah, that's fine. If the suggestion didn't seem like it was better then it isn't worth changing it. It was just an idea because I think we name other things (inside python, not YAML) with `_kwargs`. No biggie.